Rephrase the event's introduction to avoid random values in the entityID method (#246)
boostercloud/booster: docs/README.md

}
```

> An event has to know the ID of the entity it belongs to, that's why you have to implement the `entityID` method. For getting the entity ID, you could inject it directly in the event's constructor, or as a nested attribute. If your domain requires a _singleton_ entity, where there's only one instance, you can return a constant value. In the `CartPaid` example, the entity ID (`paymentID`) is injected directly.

Reviewer: I mainly wanted to correct the usage of `For` and replace `paymentID` by `cartID` (as that is the ID used in the example). But then I took the liberty to rephrase the first part. Let me know what you think 😉  
```suggestion
An event has to know the ID of the entity it belongs to and you need to implement the `entityID` method to return it. You can inject the entity ID directly in the event's constructor or as a nested attribute. If your domain requires a _singleton_ entity, where there's only one instance, you can return a constant value. In the `CartPaid` example, the entity ID (`cartID`) is injected directly.
```

Note:

> > The `entityID` method must always return the same value for the same event's instance because Booster will call it several times during its lifecycle. Otherwise, the entities' data will corrupt.

Reviewer: I would avoid saying things like `or your data will be corrupted`, as it could frighten people. Also, people might find clever ways of returning different IDs for the same event instance (not sure how). 
Here is a suggestion following what I have seen in other docs when you use something as it is not intended to be used:
```suggestion
> The `entityID` method must always return the same value for the same event's instance. Otherwise, the result of the entity reduction will be undefined.
```

Reviewer 2: Would this phrasing make more sense?

```suggestion
> The `entityID` method must always return the same value for the same event's instance. Otherwise, the result of the entity reduction will be unpredictable.
```

Reviewer: I like it better. `undefined` sounds bad too

Reviewer 3: Agree
